config.sh

```diff
@@ -12,3 +12,14 @@ function run_tests {
     python --version
     pytest -l --pyargs sklearn
 }
+
+function enable_openmp {
+    # Install OpenMP
+    brew install libomp
+    ls /usr/local/opt/libomp/include
+    ls /usr/local/opt/libomp/lib
+    export CC="clang -Xpreprocessor -fopenmp"
+    export CFLAGS="$CFLAGS -I/usr/local/opt/libomp/include"
+    export LDFLAGS="$LDFLAGS -L/usr/local/opt/libomp/lib -lomp"
+    export DYLD_LIBRARY_PATH=/usr/local/opt/libomp/lib
+}
```

Comment:
Could you please add a `disable_system_openmp` function that does `brew uninstall libomp` to be called before running the tests on the OSX wheels to make sure that the tests do not crash if libomp is not installed on the system?

Reply:
Sure. Don't know why I decided to remove it before making the PR...